gradle/libs.versions.toml

```diff
@@ -5,11 +5,18 @@ kotlin = "2.2.20"
 okhttpVersion = "5.1.0"
 spotbugs = "4.9.5"
 xpp3Version = "1.1.6"
+ktor = "3.2.1"
+slf4j = "1.7.36"
 
 [libraries]
 junit4 = { module = "junit:junit", version.ref = "junit4" }
+ktor-client-core = { module = "io.ktor:ktor-client-core", version.ref = "ktor" }
+ktor-client-logging = { module = "io.ktor:ktor-client-logging", version.ref = "ktor" }
+ktor-server-http-redirect = { module = "io.ktor:ktor-server-http-redirect", version.ref = "ktor" }
+ktor-client-mock = { module = "io.ktor:ktor-client-mock", version.ref = "ktor" }
 okhttp = { module = "com.squareup.okhttp3:okhttp", version.ref = "okhttpVersion" }
 okhttp-mockwebserver = { module = "com.squareup.okhttp3:mockwebserver3", version.ref = "okhttpVersion" }
+slf4j = { module = "org.slf4j:slf4j-android", version.ref = "slf4j" }
```

Comment:
I wonder whether we really want to define the sl4fj backend in dav4jvm? 

We could instead even use sl4fj (without defined backend) for logging in dav4jvm/ktor and say that the app that uses the library has to deal with sl4fj logging (by using the appropriate backend, which would be [slf4j-jdk14](https://www.slf4j.org/manual.html) for DAVx5)?

Reply:
Yes, thanks!